lldb/tools/lldb-dap/lldb-dap.cpp

```diff
@@ -116,6 +118,8 @@ enum LaunchMethod { Launch, Attach, AttachForSuspendedLaunch };
 /// Page size used for reporting addtional frames in the 'stackTrace' request.
 constexpr int StackPageSize = 20;
 
+void RegisterRequestCallbacks(DAP &dap);
```

Comment:
seems to be never implemented and never called?

Reply:
Removed.